app/assets/javascripts/orders.coffee

```diff
@@ -161,4 +166,6 @@ $(document).on "change", ".new-order-row .item", ->
   populateQuantity selected, quantity_element
 
 $(document).on "page:change", ->
-  addNewOrderRow() if $("#new-order-table").length > 0
+  if $("#new-order-table").length > 0
+    $.guard(".unique").using("different");
```

Comment:
You can use data guards here. Use the `data-guard="different"` attribute on the element.